tools/c7n_gcp/c7n_gcp/resources/gke.py

```diff
@@ -138,6 +152,24 @@ def get(client, resource_info):
         def _get_location(cls, resource):
             "Get the region from the parent - the cluster"
             return super()._get_location(cls.get_parent(resource))
+    def _get_location(self, resource):
+        return resource['selfLink'].split('/')[-5]
+    def augment(self, resources):
```

Comment:
same consideration here re using augment, it would be better to have a dedicated filter for these api calls.

Reply:
Updated to use a dedicated filter called server-config